lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+  attr_reader :basket, :check, :all
+
+  def initialize
+    @basket = Hash.new(0)
+  end
+
+  def add_basket(item, num)
+    basket[item] += num
+  end
+
+  def basket_summary(menu)
+    @all = menu.dishes[item]*num
+    basket.map { |item,num| "#{item} x#{num} = £#{@all}" }.join(', ')
+  end
+
+  def bill(menu)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. You can also write as `bill(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)